README.md

```diff
@@ -438,7 +438,9 @@ On Windows
 - Visual Studio 2019
 - Visual Studio 2022
 
-Other compilers or later versions may work, but they are not tested.
+Note: Visual Studio 2017 has an incomplete c++17 support. We might stop
```

Comment:
Heh.  I see @s-perron asked you to shorten this.  
But I'm curious anyway.

Reply:
This was resolved offline, but for future reference:

VS2017 is lacking support for some corner cases, example:
P1771R1 [[nodiscard]] for constructors
P0588R1 Simplifying implicit lambda capture
Might now be used right now, but since they are part of the standard, somebody could expect this to work.